include/arch/arm/arch.h

```diff
@@ -216,6 +216,12 @@ extern "C" {
 #define K_MEM_PARTITION_P_RWX_U_RWX	(P_RW_U_RW)
 #define K_MEM_PARTITION_P_RWX_U_RX	(P_RW_U_RO)
 #define K_MEM_PARTITION_P_RX_U_RX	(P_RO_U_RO)
+
+#define K_MEM_PARTITION_IS_WRITABLE(attr) \
+	((attr) & (P_RW_U_RW | P_RW_U_RO | P_RW_U_NA))
```

Comment:
Can't simply use OR operation to check here, since the P_RO_U_RO attribute will also make this statement
to be Ture.

Reply:
Done.